src/lib-components/BlockForm.vue

```diff
@@ -362,27 +369,272 @@ export default {
                 window.scrollTo(0, 0)
             }
         },
+        closeBlockForm() {
+            this.isClosed = !this.isClosed
+        },
     },
 }
 </script>
 
 <style lang="scss" scoped>
 .block-form {
+    border-radius: var(--rounded-slightly-all);
+    border-color: var(--color-primary-blue-03);
+    margin-bottom: var(--space-l);
+    max-width: $container-l-text + px;
+    padding: var(--space-xl);
+
+    @media #{$medium} {
+        max-width: 100%;
+        min-width: 600px;
+    }
+
+    @media #{$small} {
+        max-width: 100%;
+        min-width: 320px;
+    }
+
+    .formTitle {
+        @include step-1;
+        padding-bottom: 9px;
+        border-bottom: 1px solid var(--color-secondary-grey-01);
+    }
+
+    label {
+        @include step-0;
+        display: flex;
+        flex-direction: row;
+        width: 148px;
+        gap: var(--space-s);
+        margin-right: 16px;
+
+        @media #{$medium} {
+            width: 100%;
+            align-items: flex-start;
+        }
+    }
+
+    label:required {
+        @include step-0;
+        font-weight: $font-weight-medium;
+    }
+
+    ::placeholder {
+        @include step--1;
+        color: var(--color-secondary-grey-04);
+    }
+
+    .formTitleWrapper {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
```

Comment:
border goes on parent rather than the title text so that it spans full content width
`border-bottom: 1px solid var(--color-secondary-grey-02);`

Reply:
Done

![image](https://user-images.githubusercontent.com/47605309/195166894-269b8a5f-91e9-4dac-a4fc-ed15541f6fa7.png)